BUILDING.md

```diff
@@ -19,7 +19,7 @@ sudo gem update
 
 3) Some dependencies are added via carthage. Run:
 ```
-carthage update
+carthage checkout
```

Comment:
`carthage checkout` is analogous to `pod install`.
`carthage update` is analogous to `pod update`.